virttest/migration.py

```diff
@@ -578,6 +579,29 @@ def set_migratepostcopy(self, vm_name, uri=None):
             raise exceptions.TestError("Unalbe to find event {}"
                                        .format(exp_str))
 
+    def control_migrate_speed(self, vm_name, to_speed=1, opts=""):
+        """
+        Set migration speed to control the migration duration
+
+        :param vm_name: vm name
+        :param to_speed: the speed value in Mbps to be set for migration
+        :param opts: the options for virsh command
+        """
+
+        def _set_speed(vm_name, to_speed, opts=''):
```

Comment:
I assume those parameters can be referred directly from control_migrate_speed since _set_speed is inner method of control_migrate_speed

Reply:
updated

Reply:
@chunfuwen I have solved your comments. Because I have an urgent requirement for this pr and another pr depending on it to be merged asap, I requested to merge it now without your approval. If you have further comments, let me know, then I 'd like to solve them in new PR. Thanks.